ly2video/cli.py

```diff
@@ -920,6 +921,11 @@ def parseOptions():
 
     group_os = parser.add_argument_group(title='External programs')
 
+    group_os.add_argument(
+        "--windows-lilypond", dest="winLilypond",
```

Comment:
Does this have to be Windows-specific?  Why not just allow optionally specifying an arbitrary path for where to find `lilypond`, e.g. `--lilypond="C:\\lilypond\\bin\\lilypond"`?  This would allow greater flexibility on other operating systems too.